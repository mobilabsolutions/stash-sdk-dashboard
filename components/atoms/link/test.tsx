/* eslint-env jest */

import { testRender } from '../../../test_utils'
import Link from './index'

it('Link Should Render', () =>
  testRender(Link, { lable: 'Login', href: 'https://google.com' }))
