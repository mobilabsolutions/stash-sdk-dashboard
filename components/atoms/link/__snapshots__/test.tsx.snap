// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Should Render 1`] = `
<a
  className="link__A-sc-1qbu0ar-0 dGCHtZ"
  href="https://google.com"
  onClick={[Function]}
>
  <span />
</a>
`;
